mpas_analysis/test/test_generalized_reader.py

```diff
@@ -0,0 +1,100 @@
+"""
+Unit test infrastructure for the generalized_reader.
+
+Xylar Asay-Davis
+02/15/20167
+"""
+
+import pytest
+from mpas_analysis.test import TestCase, loaddatadir
+from mpas_analysis.shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+
+
+@pytest.mark.usefixtures("loaddatadir")
```

Comment:
I'm totally confused by this comment.  I just copied your `test_mpas_xarray.py` file and decorated it the same way you had done.  If this is not correct, the same problem is found in many other tests.